### PR TITLE
devops(contract): refactor to model lib

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: libs/contract/src/lib/contract/+state/contract.model.ts [hunk 1]
```diff
@@ -1,69 +0,0 @@
-
```
Comment (phpgeekfr) [inline libs/contract/src/lib/contract/+state/contract.model.ts:1]: file looks like empty but still here, can you remove it ?

File: apps/catalog/catalog/src/app/dashboard/sales/shell/shell.component.ts [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+import { ContractService } from '@blockframes/contract/contract/+state';
 import { ActivatedRoute } from '@angular/router';
 import { pluck, shareReplay, switchMap } from 'rxjs/operators';
 import { centralOrgId } from '@env';
 import { joinWith } from '@blockframes/utils/operators';
 import { MovieService } from '@blockframes/movie/+state/movie.service';
 import { IncomeService } from '@blockframes/contract/income/+state';
 import { Intercom } from 'ng-intercom';
+import { Sale, contractStatus } from '@blockframes/model';
+
```
Comment (phpgeekfr) [inline apps/catalog/catalog/src/app/dashboard/sales/shell/shell.component.ts:12]: please remove extra lines

File: apps/backend-functions/src/templates/mail.ts [hunk 3]
```diff
@@ -6,16 +6,14 @@
 import { supportEmails, appUrl, e2eMode } from '../environments/environment';
 import { EmailRequest, EmailTemplateRequest } from '../internals/email';
 import { templateIds } from '@blockframes/utils/emails/ids';
-import { EventEmailData, getMovieEmailData, getOfferEmailData, MovieEmailData, OrgEmailData, UserEmailData } from '@blockframes/utils/emails/utils';
 import { RequestDemoInformations } from '../data/types';
-import { PublicUser, OrganizationDocument, PublicOrganization, MovieDocument } from '@blockframes/model';
+import { PublicUser, OrganizationDocument, PublicOrganization, MovieDocument, createMailContract, ContractDocument } from '@blockframes/model';
```
Comment (phpgeekfr) [inline apps/backend-functions/src/templates/mail.ts:10]: can you check if you can add `type` like this `import type` ?

File: libs/utils/src/lib/emails/utils.ts [hunk 4]
```diff
@@ -11,15 +11,14 @@ import {
   User,
   PublicUser,
   OrganizationDocument,
-  orgName
+  orgName,
+  ContractDocument
 } from '@blockframes/model';
```
Comment (phpgeekfr) [inline libs/utils/src/lib/emails/utils.ts:16]: same

File: apps/backend-functions/src/templates/mail.ts (hunk 3 above)
Comment (phpgeekfr) [reply apps/backend-functions/src/templates/mail.ts:10]: can you try with 
```
import  { PublicUser, OrganizationDocument, PublicOrganization, MovieDocument, createMailContract } from '@blockframes/model';
import type {  ContractDocument } from '@blockframes/model';
```

File: libs/utils/src/lib/emails/utils.ts (hunk 4 above)
Comment (KvnMrs) [reply libs/utils/src/lib/emails/utils.ts:16]: it doesn't work too, if my syntax that i showed you is correct

**Pull request merged**